src/testdrive/src/parser.rs

```diff
@@ -141,6 +130,21 @@ fn parse_builtin(line_reader: &mut LineReader) -> Result<BuiltinCommand, InputEr
     })
 }
 
+/// Validate that the string is an allowed variable name (lowercase letters, numbers and dashes)
+pub fn validate_ident(name: &str) -> Result<(), String> {
+    lazy_static! {
+        static ref VALID_KEY_REGEX: Regex = Regex::new("^[a-z0-9\\-]*$").unwrap();
```

Comment:
We should allow dots here, as elsewhere we allow/have hierarchical variable names , e.g. `kafka-ingest.iteration`

Reply:
Oh nevermind, you are just moving the eixsting regexp around.